pkg/image/vulnerability_scan_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package image_test
+
+import (
+	"context"
+	"os"
+	"path"
+
+	shipwrightv1alpha1 "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
+	"github.com/shipwright-io/build/pkg/image"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("VulnerabilityScanning", func() {
+
+	Context("For an index", func() {
+		vulnOptions := shipwrightv1alpha1.VulnerabilityScanOptions{
+			Enabled: true,
+		}
+		cwd, err := os.Getwd()
+		Expect(err).ToNot(HaveOccurred())
+		directory := path.Clean(path.Join(cwd, "../..", "test/data/images/vuln-image-in-oci"))
+
+		It("runs the image vulnerability scan", func() {
+			//Run vulnerability Scan
+			vulns, err := image.RunVulnerabilityScan(context.TODO(), directory, vulnOptions)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(vulns).NotTo(BeNil())
+			Expect(len(vulns)).Should(BeNumerically(">", 0))
```

Comment:
Nit: Gomega has some very specific matchers at one's disposal.
```suggestion
			Expect(vulns).ToNot(BeEmpty())
```